src/server/graphql/mutations/helpers/makeRetroGroupTitle.js

```diff
@@ -0,0 +1,81 @@
+/*
+ * Takes a guess at what the cards are talking about.
+ * If that fails, just gives them a generic name
+ */
+import getRethink from 'server/database/rethinkDriver';
+import extractTextFromDraftString from 'universal/utils/draftjs/extractTextFromDraftString';
+import language from '@google-cloud/language/src/index';
+import sendSentryEvent from 'server/utils/sendSentryEvent';
+
+// This is how we know we caught all the important words. Lower catches more
+const SALIENT_THRESHOLD = 0.85;
+const MIN_ENTITIES = 2;
+const MAX_CHARS = 40;
+
+const getSmartEntities = async (reflections) => {
+  const content = reflections
+    .map((reflection) => extractTextFromDraftString(reflection.content))
+    .join('\n');
+  const client = new language.LanguageServiceClient({keyFilename: '../../../../../.googlecloudkey.json'});
```

Comment:
Is there a way to do this with an environment variable instead?

Reply:
no because google wants it to be as painful as possible so you store the file on THEIR server & pay them $$. But if you ask them, they'll tell you it's so you can use "service accounts". it's hogwash.

Reply:
Ugh. Ok. Let me think about a strategy for this. I might want to augment this branch.